src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -3522,6 +3613,33 @@ mgp_error mgp_graph_aggregate_over_text_index(mgp_graph *graph, const char *inde
   });
 }
 
+mgp_error mgp_graph_has_vector_index(mgp_graph *graph, const char *index_name, int *result) {
+  return mgp_error::MGP_ERROR_NOT_YET_IMPLEMENTED;
+}
+
+mgp_error mgp_graph_search_vector_index(mgp_graph *graph, const char *index_name, mgp_list *search_query,
```

Comment:
Let's check if PMR data structures should be used here because this is called with `CALL`, probably consult with @Ignition, not required under this PR. Before making this stable feature, we should be confident about that.